src/networkremote/outgoingdatacreator.h

```diff
@@ -83,6 +87,8 @@ class OutgoingDataCreator : public QObject {
   void ResultsAvailable(int id, const SearchProvider::ResultList& results);
   void SearchFinished(int id);
 
+  void SendListFiles(QString relative_path);
```

Comment:
`const QString&`

Reply:
Indeed, the IncomingDataParser and the OutgoingDataCreator are living in the thread.
but the signal just above wasn't using cont ref.
```
    connect(incoming_data_parser_.get(),
            SIGNAL(DoGlobalSearch(QString, RemoteClient*)),
            outgoing_data_creator_.get(),
            SLOT(DoGlobalSearch(QString, RemoteClient*)));

    connect(incoming_data_parser_.get(), SIGNAL(SendListFiles(QString)),
            outgoing_data_creator_.get(), SLOT(SendListFiles(QString)));
```
I guess I'll update both ;)

Reply:
hum in fact I'd rather keep it as a copy as I may remove the first slash
`if (relative_path.startsWith("/")) relative_path.remove(0, 1);`

Reply:
so I've let it